test/acceptance/features/bindAppToService.feature

```diff
@@ -689,6 +697,11 @@ Feature: Bind an application to a service
                 name: binding-request-remove-service
             spec:
                 services:
+                application:
+                    name: gen-app-a-s-e
+                    group: apps
+                    version: v1
+                    resource: deployments
```

Comment:
why this change? even without the application reference, the service binding was invalid.

Reply:
This a valid negative scenario. See the title: `Removing service from services field from existing serivce binding is not allowed`